lib/gui/app.js

```diff
@@ -176,6 +176,11 @@ app.run(() => {
     ].join(' '));
 
     windowProgress.set(currentFlashState.percentage);
+
+    const DASH_UNICODE_CHAR = '\u2013';
+    const flashProgress = messages.info.flashProgress(currentFlashState);
+    const title = `${packageJSON.displayName} ${DASH_UNICODE_CHAR} ${flashProgress}`;
+    windowProgress.setTitle(title);
```

Comment:
> EDIT: i.e. maybe FlashController.getProgressButtonLabel() and windowProgress.set() should both now call a new shared method.

Yeah, that sounds like the way to go. Also, we should make this window progress function accept a "progress state" object (the one coming from `etcher-image-write`, which has the percentage, type, speed, etc) rather than just a percentage, so the function has enough context to know if the percentage is avoid writing or validation.